tests/phpunit/includes/install.php

```diff
@@ -42,6 +42,15 @@
 require_once ABSPATH . 'wp-admin/includes/upgrade.php';
 require_once ABSPATH . 'wp-includes/class-wpdb.php';
 
+if ( $wpdb->last_error ) {
+	printf(
```

Comment:
Was there a reason why this is preferred over throwing an `Exception` as suggested initially on the ticket?